web/react/stores/browser_store.jsx

```diff
@@ -8,7 +8,7 @@ function getPrefix() {
 }
 
 // Also change model/utils.go ETAG_ROOT_VERSION
-var BROWSER_STORE_VERSION = '.3';
+var BROWSER_STORE_VERSION = '.4';
```

Comment:
Actually this was intentional because I didn't want merge conflicts. I figured we would change it after all these PRs go in. (I thought they might change it as well)